docs/source/concepts/ledger-model/ledger-exceptions.rst

```diff
@@ -144,13 +144,14 @@ More generally, two transactions are equivalent if:
 
 - The transactions have the same "before-after" relation between their actions.
 
-- The transactions have the same set of rolled back actions. That is, they have
-  the same set of actions directly or indirectly under a rollback node.
+- The transactions have the same set of "rollback roots".
```

Comment:
The name seems quite confusing. A rollback root is not a rollback node. It’s the child of a rollback node. Maybe “rollback children” (but not descendants) would be clearer?

Reply:
Ah, yeah. I wasn't sure about the name. "Rollback child"/"rollback children" works.

Reply:
Changed. How's this?